spec/features/user/dashboard_spec.rb

```diff
@@ -10,18 +10,73 @@
     expect(page).to have_no_content("GitHub")
   end
 
-  it 'can visit dashboard and see GitHub repos' do
+  it 'can visit dashboard and see GitHub repos', :js do
```

Comment:
Not entirely sure what `:js` entails but it allowed us to check the `current_url` without it hitting the routes